### ISSUE TITLE
On the walk at the make-out spot it can take an ridiculous amount fo time to find anything

### ISSUE THREAD

**Issue by Forget-About-Me2** [FamTroder/FamMel#167]
Both the beach and the dark yard, if you have bad luck you just have to keep clicking and nothing else. It's a bit boring. maybe increase chances + more options while walking?

**Comment by M0nkey5ee** [comment]
Can help you with building more options if you want to - great work by the way, love the whole doc!

**Comment by Forget-About-Me2** [comment]
yeah, I don't have any real ideas. Adding more things you can find I guess. Maybe coming accross other people (maybe someone who is desperate to pee/clearly peeing). Also potential to find other locations aside form the hot tub/beach (I'm thinking maybe like beach party or like a camp fire you can join) But not sure what that could offer in terms of game play options

**Comment by M0nkey5ee** [comment]
But these already sound great?

1. Coming across other girls was one of the most undervalued feature in the jennygame (?) if I recall correctly! Like your main girl and one other have to navigate their desperate final together was and still sounds like a REALLY great idea!
2. Beach Party and Camp fire also sound awesome! Maybe a little festival of sorts? If you build the code I can take care of the story elements, my own test right now goes into a similar direction but I only make small progress right now.
3. Maybe just a romantic cliff / cave / secluded part of the beach which gets unlocked when you bring a tent with you from your car? Could also make a good scene I think.

**Comment by Forget-About-Me2** [comment]
Those were really just me spitballing. I just made them up as I was typing.
1. I don't know if I ever played the jennygame, I know I played the diana game and another one I think. But I can't find the files for those anymore. I think for this just adding some scenes with other people would be enough. Actually adding more girls with bladders that need to be kept track of his a bit hard at this point as there is no support for . Which is more something to be dealt with if #103  ever gets implemented.
2. The location ideas sound nice. But we would need some ideas on what is special about them gameplay wise (idk beach party  could maybe somehow resulting in other npc encouraging you or the girl holding it without you having to do much.) Fstival I was maybe thinking the idea of #104 to have like an omo festival kind of idea? These need to be worked out a bit more to have a good plan.
3. Maybe, would need to convince her to go camping. Also would need a way to get a tent as I think this shouldn't be in the shop at the beginning (maybe you end up meeting someone somewhere who offers it either for money or maybe a sexual favour even?)
